_data/changelogs/utilities.yml

```
@@ -9,7 +9,7 @@ items:
     githubPr: 5209
     githubRepo: uswds
     versionUswds: 3.5.0
-  - date: NNN-NN-NN
+  - date: 2023-05-26
```

Should these be the same release date as other 3.5.0 issues? [3.5.0](https://github.com/uswds/uswds/releases/tag/v3.5.0) was released on June 9. 
```suggestion
  - date: 2023-06-09
```